### ISSUE TITLE
Video Comparison Not Working

### ISSUE THREAD

**Issue by triatomic** [staxrip/staxrip#1258]
Using video comparison on mp4 videos doesn't work

Pressing video comparison doesn't work on some videos will just output a error message:

```
LSMASHVideoSource: wrong format.
(C:\Users\msi\AppData\Local\Temp\fb947a4e-a3ce-45e0-b4f4-011dc9a46aa8.avs, line 4)

DivideByZeroException (v2.36.0)

Attempted to divide by zero.

System.DivideByZeroException: Attempted to divide by zero.
   at System.Decimal.FCallDivide(Decimal& d1, Decimal& d2)
   at System.Decimal.Divide(Decimal d1, Decimal d2)
   at StaxRip.DirectFrameServer.get_FrameRate() in D:\Code\Github\staxrip\staxrip\Source\Video\FrameServer.vb:line 37
   at StaxRip.VideoComparisonForm.VideoTab.TrackBarValueChanged() in D:\Code\Github\staxrip\staxrip\Source\Forms\VideoComparisonForm.vb:line 478
```

Info:
- StaxRip version 2.36
- Nvidia v546.17
- Windows 11 22h2 (22621.3155)

**Comment by Dendraspis** [comment]
Can you post the media info of that mp4 file?

**Comment by Ding-adong** [comment]
I was going to make the same report as @triatomic. I opened a mp4 file that worked on previous version but not the latest and got same error as above.

**Comment by triatomic** [comment]
> Can you post the media info of that mp4 file?

this happens for all mp4 files; but if i rename the .mp4 video to .mkv it works fine.

**Comment by johnlee1st** [comment]
I experimented with maybe around a dozen different StaxRip versions, and the results say that the _latest_ version whose Video Comparison works without this bug is **2.30.0**.

**Comment by Dendraspis** [comment]
Meanwhile I have tested a lot of different scenarios, but I cannot reproduce it...

<img width=400 src=https://github.com/staxrip/staxrip/assets/50659978/0f7eff94-4808-44df-8969-5cf71bfe1076>

So can anybody confirm, that using `v2.38.7` with fresh/new settings works also for you?

**Comment by Dendraspis** [comment]
Great, right after I have posted it, I had an idea...
Indeed the media info or a sample would have been very helpful to solve the problem within 5min.

So it should be fixed with `v2.39`...
